### DISCUSSION TITLE
Valheimplus on Synology

### DISCUSSION THREAD

**Discussion by zorly57** [lloesche/valheim-server-docker#94] [General]
Hi,
Is it possible to use the Valheim Plus on Synology?
The docker is up to date and in the environment I add VALHEIM_PLUS true but nothing changes, so I add in Volume add folder the line / Valheim / config / valheimplus which added the folder but empty.

Thank you

**Comment by InB4DevOps** [comment]
Did you use the dev tagged image?

**Reply by zorly57** [reply]
Sorry I had to be away, no I did not try but I test immediately. Thank you

**Reply by TheMoonMan0** [reply]
did you resolve this issue? I'm having the same problem.

**Reply by adaMATTium** [reply]
Anyone running into this issue, check your environmental entry for VALHEIM_PLUS. I copy-pasted it from Github and it introduced a  tab into the field. I didn't notice it until I was poring through everything and caught it in the JSON. After I removed the space, it worked perfect.

**Comment by lloesche** [comment]
@zorly57 ValheimPlus support is now in the main branch so you can use the `:latest` tag again.

**Reply by TheMoonMan0** [reply]
are there any updates on this or am I just doing something wrong?

Thx, I love your work.

**Comment by Fabninja** [comment]
I'm having the same issue actually. I just entered VALHEIM_PLUS=true and nothing about it even shows up in the log file. I'm using the latest tag.

![Screenshot 2021-04-07 132225](https://user-images.githubusercontent.com/75227155/113929058-70554800-97a4-11eb-8412-a0417493aaeb.png)

**Reply by lloesche** [reply]
@Fabninja best create an issue and attach the complete log file there, then we'll be able to figure out what's going on.

**Reply by Fabninja** [reply]
Ok I just did. Thanks for your support!
